CHANGELOG.next.asciidoc

```diff
@@ -192,6 +192,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix panic in Redis Key metricset when collecting information from a removed key. {pull}13426[13426]
 - Fix module-level fields in Kubernetes metricsets. {pull}13433[13433] {pull}13544[13544]
 - Fix reporting empty events in cloudwatch metricset. {pull}13458[13458]
+- Fix `docker.cpu.system.pct` calculation. {pull}13691[13691]
```

Comment:
Ok, but we could mention under what circumstances this could happen, or what are we doing, e.g.
```suggestion
- Fix `docker.cpu.system.pct` calculation by using the reported number online cpus instead of the number of metrics per cpu. {pull}13691[13691]
```